src/AccessionTables/components/Location.js

```diff
@@ -3,12 +3,21 @@ import PropTypes from 'prop-types';
 
 import { EmptyMessage } from '@folio/stripes/components';
 
-export const Location = ({ location }) => (
-  location
-    ? <>{location.name} ({location.code})</>
-    : <EmptyMessage />
-);
+export const Location = ({ location, label }) => {
+  if (!location?.id) return <EmptyMessage />;
```

Comment:
Are there any cases when a `location ` cannot have an ID? I think we can just check `if (!location)`

Reply:
I've made this for the case when the `location` is somehow defined, but not the object with `id`.
Maybe it's over-engineering, IDK )

Reply:
Okay, since the check for location is present anyway )